postgres/portal_app.go

```diff
@@ -0,0 +1,100 @@
+package postgres
```

Comment:
Consider renaming to: `postgres/grove/app.go` or `postgres/grove/portal/app.go` or `postgres/grove/porta_app.go`

If we have a dedicated `grove` package, someone could just `copy -r` the directory and do what they want.